docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -7,140 +7,39 @@ title: Publish a plugin
--rw-r--r--        808 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/module.js.LICENSE.txt
--rw-r--r--     487395 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/module.js.map
--rw-r--r--       1616 bX defN 22-Mar-24 23:23 grafana-clickhouse-datasource/plugin.json
-22 files, 92516655 bytes uncompressed, 34319591 bytes compressed:  62.9%
-```
-
-## Publish your plugin on Grafana.com
-
-The best way to share your plugin with the world is to publish it on [Grafana Plugins](/plugins).
-By having your plugin published on Grafana.com, more users will be able to discover your plugin.
-
-Before you submit your plugin, we ask that you read our guidelines and frequently asked questions.
-
-### Guidelines
-
-To speed up the time it takes to review your plugin:
-
-- Get familiar with our plugin [publishing and signing criteria]({{< relref "./publishing-and-signing-criteria" >}})
-- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
-- Read our [6 tips for improving your Grafana plugin before you publish](/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
-- Refer to [plugin-examples](https://github.com/grafana/grafana-plugin-examples) to review best practices for building your plugin.
+# Publish or update a plugin
 
-### Frequently Asked Questions
+You've just built your first plugin and [packaged it](package-a-plugin.md), and now you want to share it with the world.
+In this guide, you'll learn how to share your plugin with others.
 
-**Do I need to submit a private plugin?**
-
-- No. Please only submit plugins that you wish to make publicly available for the Grafana community.
-
-**How long does it take to review my submission?**
-
-- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin.
-
-**Can I decide a date when my plugin will be published?**
+The best way to share your plugin with the world is to publish it in the [Grafana plugin catalog](/plugins).
+By having your plugin published on Grafana.com, more users will be able to discover your plugin.
 
-- No. We cannot guarantee specific publishing dates, as plugins are immediately published after a review based on our internal prioritization.
+## Before you begin
 
-**Can I see metrics of my plugin installs, downloads or usage?**
+Before you begin to publish a plugin, it needs to be built and packaged. 
 
-- No. We don't offer this information at the moment to plugin authors.
+To build a plugin from source, Grafana creates a `dist` directory that contains the production build, or _plugin assets_, for your plugin.
 
-**How can I update my plugin's catalog page?**
+When the Grafana server starts, it recursively looks in the plugin directory for any directory that contains a `plugin.json` file, and then it tries to load the plugin assets in the same directory.
```

Comment:
I don't see this sentence fitting in this context.

Reply:
This PR is now closed. This issue is fixed in PR 67164.